CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-0.17.0 / TBC
+0.16.2 / 2017-10-3
 ===================
-- Update all available npm dependencies
+- Update ADR guidance/superseded mongo ADR
```

Comment:
I wouldn't bother including a note about this change. No functionality has changed, just that documentation. As a third party I'm only interested in changes of interest and that piece of documentation isn't interesting as the functional change has been already.